src/components/Select/MultiSelect.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import ChevronDownIcon from '@moda/icons/chevron-down-12';
+import ChevronUpIcon from '@moda/icons/chevron-up-12';
+import ExitIcon from '@moda/icons/exit-16';
+import { SelectableOption } from '.';
+import { SelectOptions } from './SelectOptions';
+import { useSelect } from './useSelect';
+import './MultiSelect.scss';
+
+type Props = Omit<React.HTMLAttributes<HTMLDivElement>, 'defaultValue' | 'value' | 'onChange'> & {
+  defaultValue?: string[] | undefined;
+  disabled?: boolean;
+  idRef?: string;
+  name?: string;
+  placeholder?: string;
+  onChange?: (value: string[]) => void;
+  onRemoveSelectedItem: (value: string) => void;
```

Comment:
I think this is redundant cause when you remove an item `onChange` is called with the new list of values right?

Reply:
correct good call